corekit/src/main/java/com/midtrans/sdk/corekit/models/TransactionResponse.java

```diff
@@ -143,6 +143,9 @@
     private String qrCodeUrl;
     @SerializedName("deeplink_url")
     private String deeplinkUrl;
+    //currently for shopeepay
+    @SerializedName("qris_url")
```

Comment:
can in the future this applies to other QR based payment as well?

Reply:
yes, it will be used for other QR payment as well, but currently only applies for shopeepay to differentiate it with existing gopay

Reply:
can you update the comment then explaining that it's for non gopay QR, not specific to shopeepay

Reply:
okay